testsuite/integration-arquillian/tests/base/src/main/java/org/keycloak/testsuite/util/OAuthClient.java

```diff
@@ -725,7 +725,7 @@ public URI getCurrentUri() {
 
     public Map<String, String> getCurrentQuery() {
         Map<String, String> m = new HashMap<>();
-        List<NameValuePair> pairs = URLEncodedUtils.parse(getCurrentUri(), Charset.forName("UTF-8"));
+        List<NameValuePair> pairs = URLEncodedUtils.parse(getCurrentUri(), "UTF-8");
```

Comment:
There is only `public static List <NameValuePair> parse(final URI uri, final String charset)` in [httpclient-4.5.2.Final URLEncodedUtils](https://github.com/apache/httpcomponents-client/blob/4.5.2/httpclient/src/main/java/org/apache/http/client/utils/URLEncodedUtils.java#L90) so it seems we are not able to prevent repeating looups for now. I can replace with `StandardCharsets.UTF_8.name()` if your prefer it.